lib/jwt/jwt.go

```diff
@@ -220,7 +219,7 @@ func (k *Key) Verify(p VerifyParams) (*Claims, error) {
 	expectedClaims := josejwt.Expected{
 		Issuer:   k.config.ClusterName,
 		Subject:  p.Username,
-		Audience: jwt.Audience{p.URI},
+		Audience: josejwt.Audience{p.URI},
```

Comment:
Why is `josejwt` preferred over `jwt` here?  Looking at the imports it seems like it could have gone either way?